pkgs/development/python-modules/pyisbn/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, hypothesis, lib, pytest, pytest-randomly
+, pytest-cov }:
```

One input per line make the diffs in the future more clear.

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, hypothesis
, pytest
, pytest-randomly
, pytest-cov
}:
```